src/server/journal/streamer.cc

```diff
@@ -6,27 +6,57 @@
 
 #include <absl/functional/bind_front.h>
 
+#include <chrono>
+#include <system_error>
+
 #include "base/logging.h"
 
 namespace dfly {
 using namespace util;
 
-void JournalStreamer::Start(io::Sink* dest) {
+void JournalStreamer::PeriodicPing::MaybePing(bool allow_await) {
+  const auto now = std::chrono::system_clock::now();
+  const auto elapsed = std::chrono::duration_cast<std::chrono::seconds>(now - start_time_);
+  if (elapsed > kLimit) {
```

Comment:
where kLimit is defined?

Reply:
static member of `JournalStreamer::PeriodicPing`